apps/example-solid-start/src/routes/api/auth/[...solidauth].ts

```diff
@@ -0,0 +1,26 @@
+import {
+  SolidAuth,
+  type ISolidAuthHandlerOpts,
+} from "@solid-auth/next/handler";
+import GitHub from "@auth/core/providers/github";
+import { serverEnv } from "~/env/server";
+import { type APIEvent } from "solid-start";
+
+export const authOpts: ISolidAuthHandlerOpts = {
+  providers: [
+    GitHub({
+      clientId: serverEnv.GITHUB_ID,
+      clientSecret: serverEnv.GITHUB_SECRET,
+    }),
+  ],
+  debug: false,
+};
+
+const handler = SolidAuth(authOpts);
```

Comment:
Let's make sure that the package allows this, it's cleaner.:

```suggestion
export const { GET, POST } = SolidAuth(authOpts);
```

Reply:
this might not always work, i used functions because functions get hoisted, 
for some reason when i used const with the node adapter then trying to build the package it threw "cannot use GET before initialization"

Reply:
done